Formula/arx-libertatis.rb

```diff
@@ -25,6 +25,23 @@ class ArxLibertatis < Formula
   def install
     args = std_cmake_args
 
+    # The patches for these aren't straightforward to backport because of
+    # other changes; these minimal inreplaces get it building.
+    # HEAD is fine, and the next stable release will contain these changes.
```

Comment:
Agreed. Development is clearly very active, but with four years between releases I would really like to see another.